kindle.py

```diff
@@ -576,6 +585,13 @@ def download_books(self, start_index=0, filetype="EBOK"):
         help="If only generate kindle readme stats",
     )
 
+    parser.add_argument(
+        "--list",
```

Comment:
--list-only or --list-books maybe nicer?

Reply:
my idea is list-only , but i think type less a few charactors may be more user friendly 🤔

Reply:
I prefer --list-only
--list mostly has other meaning